etc/kayobe/ansible/stackhpc-cloud-tests.yml

```
@@ -60,38 +66,124 @@
             file: "{{ kayobe_env_config_path }}/kolla/passwords.yml"
             name: kolla_passwords
 
-        - name: Run StackHPC Cloud tests
+        # Monitoring tests should run once, executed on the host in the
+        # tempest_runner group.
+        - name: Check for StackHPC Cloud monitoring tests
+          ansible.builtin.stat:
+            path: "{{ repo_tmpdir.path }}/stackhpc_cloud_tests/monitoring"
+          register: stackhpc_cloud_monitoring_tests
+        - name: Run StackHPC Cloud monitoring tests
```

By convention, we keep a newline between tasks, could you separate these out? (there are more instances, I'm just pointing out the first)
```suggestion
          register: stackhpc_cloud_monitoring_tests

        - name: Run StackHPC Cloud monitoring tests
```